test/parallel/forall/task-private-vars/count-tpvs.chpl

```diff
@@ -0,0 +1,24 @@
+// Ensure no extra TPVs get created.
+
+config const numTasks = 7;
```

Comment:
I suspect this test will fail on low core count machines. Instead of setting numTasks like this, I'd set `--dataParTasksPerLocale=2` in an execopts file (we do this for bale tests, and it has proved to be the most portable --https://github.com/chapel-lang/chapel/commits/master/test/studies/bale/histogram/histo-atomics.execopts)

Reply:
And actually since it's an internal assert you probably don't need the execopts. 

Can probably do:

```chpl
const hereTasks = here.maxTaskPar;
const totalTasks = + reduce Locales.maxTaskPar;

forall 1..hereTasks...
assert hereTasks..


var A = newBlockArr(1..totalTasks)...
forall a in A

assert totalTasks...
```